charters/DataProcessingService/charter.md

```diff
@@ -0,0 +1,69 @@
+# [Data Processing Service](mailto:pipelines-team@data.humancellatlas.org)
+
+## Description
+
+The Data Processing Service is a service that executes data processing pipelines containing scientific analysis on scientific data in the Data Coordination Platform (DCP).  
+
+## Objectives
+
+The Data Processing Service develops the infrastructure to run pipelines reliably and at-scale for the DCP. The team implementing the service also operates the pipeline infrastructure for the Human Cell Atlas project. 
+
+## Definitions
+
+__pipeline:__ A collection of one or more functional tasks that operate on input data and, from that, make an output that is transformed input data, or features generated to interpret the input data.
```

Comment:
@TimothyTickle thank you for adding this. Still, this definition could apply to other software projects in DCP. Take the data browser, for example:

* `A collection of one or more functional tasks that operate on input data` - assembling bundles into specimens and indexing them
* `make an output that is transformed input data` - transforms bundles into specimen records for the data browser
* `or [make] features generated to interpret the data` - one could argue that facets used for searching the data are extracted "features"

This will likely be confusing to uninitiated readers.

This definition/charter should provide sufficient information to answer the question: what makes data processing pipelines in the green box unique from other software projects in DCP?

Reply:
Per Matt's comment, I think this definition could be clarified by answering why these pipelines are important or what they typically contain.